src/app/secure-data-environment/endpoints/organisation.endpoints.ts

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+import { Organisation, OrganisationMember } from '../resources/organisation.resources';
+import { Observable } from 'rxjs';
+import { ISdeRestHandler } from '../sde-rest-handler.interface';
+import { Uuid } from '@maurodatamapper/mdm-resources';
+import { SdeApiEndPoints } from './endpoints.dictionary';
+
+export class OrganisationEndpoints {
```

Comment:
Do we want one example of a command endpoint that performs a POST and creates/updates an entity?

Reply:
The mdm-explorer will not be posting any organisation related data. We will need to add this example in to the SDE-Admin-UI version of this task. It is scoped to do posts. See: https://github.com/MauroDataMapper/sde-admin-ui/issues/12